### PR TITLE
Office Hours 2019-09-03

### PR DESCRIPTION
**Pull request by directionless**
Office hours!

@theopolis, @smjert, @mike-myers-tob you probably all want to review this

### CODE REVIEW

File: docs/office-hours/20190903_meeting_notes.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+What if we limited the supported platforms? Would this let us ship faster? What would our messaging here be? Does anyone use CentOS6. We believe that we are as stable as in prior releases. But we understand more about third party libraries, and we think there are places we can improve for older versions of libc.
+
+More people to help update third-party libs would help. Stefano has done about seven libraries. We have about 40. Not all libraries are equal.
+
+#### How do we move forward?
+
+We **will** merge the custom toolchain work. This needs a small amount of doc work, and binaries somewhere. https://github.com/osquery/osquer
+
+_Then_ we will tackle the third-party dependencies one at a time, getting support for older libc into their configuration files. This work will be done a third-party lib at a time.
```
Comment (directionless) [inline docs/office-hours/20190903_meeting_notes.md]: Do folks want me to make tickets for each third party lib? I can automate it, it's easy, but it is 40 new tickets

File: docs/office-hours/20190903_meeting_notes.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+There are some concerns about merging the toolchain work. As it stands, it probably breaks build and runtime on some older platforms. But, the current build uses the OS toolchain, which is also broken on those platforms. So merging is not worse.
+
+What if we limited the supported platforms? Would this let us ship faster? What would our messaging here be? Does anyone use CentOS6. We believe that we are as stable as in prior releases. But we understand more about third party libraries, and we think there are places we can improve for older versions of libc.
+
+More people to help update third-party libs would help. Stefano has done about seven libraries. We have about 40. Not all libraries are equal.
+
+#### How do we move forward?
+
+We **will** merge the custom toolchain work. This needs a small amount of doc work, and binaries somewhere. https://github.com/osquery/osquer
```
Comment (directionless) [inline docs/office-hours/20190903_meeting_notes.md]:
```suggestion
We **will** merge the custom toolchain work. This needs a small amount of doc work, and binaries somewhere. https://github.com/osquery/osquery/issues/5609
```

File: docs/office-hours/20190903_meeting_notes.md (hunk 1 above)
Comment (Smjert) [reply docs/office-hours/20190903_meeting_notes.md]: My take is that if we expect people to subdivide work, and do this gradually, I would say that we need them to be separate.
At the same time, we should though have a single issue which covers CentOS 6 support (or similar) that collects them all, with a check list for each of them.
Much like https://github.com/osquery/osquery/issues/5652.

File: docs/office-hours/20190903_meeting_notes.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+What if we limited the supported platforms? Would this let us ship faster? What would our messaging here be? Does anyone use CentOS6. We believe that we are as stable as in prior releases. But we understand more about third party libraries, and we think there are places we can improve for older versions of libc.
+
+More people to help update third-party libs would help. Stefano has done about seven libraries. We have about 40. Not all libraries are equal.
+
+#### How do we move forward?
+
+We **will** merge the custom toolchain work. This needs a small amount of doc work, and binaries somewhere. https://github.com/osquery/osquery/issues/5609
+
+_Then_ we will tackle the third-party dependencies one at a time, getting support for older libc into their configuration files. This work will be done a third-party lib at a time.
```
Comment (directionless) [inline docs/office-hours/20190903_meeting_notes.md]:
```suggestion
_Then_ we will tackle the third-party dependencies one at a time, getting support for older libc into their configuration files. This work will be done a third-party lib at a time. https://github.com/osquery/osquery/issues/5745
```

File: docs/office-hours/20190903_meeting_notes.md (hunk 1 above)
Comment (directionless) [reply docs/office-hours/20190903_meeting_notes.md]: Works for me. https://github.com/osquery/osquery/issues/5745

File: docs/office-hours/20190903_meeting_notes.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
+# osquery Office Hours 2019-09-03
+
```
Comment (directionless) [inline docs/office-hours/20190903_meeting_notes.md]:
```suggestion

YouTube Recording: https://youtu.be/urMFR1ICFbU
```

File: docs/office-hours/20190903_meeting_notes.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+Let's try it. @directionless will click some buttons to set it up.
+
+## Finance / Fundraising
+
+This is https://github.com/osquery/foundation/issues/15
+
+The readthedocs people reached out to us about profit sharing, which means we need a bank account. The Linux Foundation pointed us at Community Bridge, a platform they're building out.
+
+seph went through the signup procress, and we are now live at https://funding.communitybridge.org/projects/osquery It all looks a bit rough right now. But, we can probably take donations in, and the TSC can claim expenses.
```
Comment (burdzwastaken) [inline docs/office-hours/20190903_meeting_notes.md]:
```suggestion
seph went through the signup process, and we are now live at https://funding.communitybridge.org/projects/osquery It all looks a bit rough right now. But, we can probably take donations in, and the TSC can claim expenses.
```

**Comment by theopolis** [comment]
Thanks for keeping minutes, this is awesome!

**Pull request merged**